backend/tracim_backend/models/context_models.py

```diff
@@ -814,6 +814,18 @@ def __init__(
         self.description = description
 
 
+class ContentNamespaceUpdate(object):
+    """
+    Content namespace update model
+    """
+
+    def __init__(
+        self,
+        content_namespace: Optional[str] = None,
+    ) -> None:
+        self.content_namespace = content_namespace
+
+
```

Comment:
Why `content_namespace` attribute is optional ? As I can see in `ThreadController.update_thread_namespace` route, specially in `api.set_content_namespace` call, the namespace is a required. It appears to me to be a required value.

Reply:
yes, indeed I will set the namespace as required